docs/source/support/faq.rst

```diff
@@ -29,7 +29,7 @@ NuCypher currently has two testnets available, each of them with different goals
 Q: How long is a period?
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
-1 period is 24 hours. Periods begin at midnight UTC.
+1 period is 7 days, and periods start at midnight UTC.
```

Comment:
I guess better to add which day midnight

Reply:
A specific day would be great. Presumably this depends on when we upgrade the contract; is there a plan to do the upgrade on a specific day?

Reply:
I think it should be unrelated to upgrade day, it's day when current period division by 7 without a remainder